src/features/device-state/routes/state-patch-v3.ts

```diff
@@ -103,11 +98,13 @@ const fetchData = async (
 					belongs_to__application: {
 						$select: 'uuid',
 					},
+					reports__device_metrics_record: {},
 				},
 			},
 		})) as Array<
 			Pick<Device, 'id' | 'uuid'> & {
 				belongs_to__application: Array<Pick<Application, 'uuid'>>;
+				reports__device_metrics_record: DeviceMetricsRecord[];
```

Comment:
We don't really care to fetch any of the rest columns.
```suggestion
					reports__device_metrics_record: {
						$select: 'id'
					},
				},
			},
		})) as Array<
			Pick<Device, 'id' | 'uuid'> & {
				belongs_to__application: Array<Pick<Application, 'uuid'>>;
				reports__device_metrics_record: reports__device_metrics_record: [Pick<DeviceMetricsRecord, 'id'>?];
```